sdk/storage/azure-storage-file-datalake/tests/test_file_async.py

```diff
@@ -98,10 +94,23 @@ async def _create_file_and_return_client(self, directory="", file=None):
         await file_client.create_file()
         return file_client
 
+    def _is_almost_equal(self, first, second, delta):
```

Comment:
Same feedback here, just making a note so that I remember to change it in both places (if we decide on a name change)

Reply:
Same response. Looks good.